### PR TITLE
Change all references to the glade/p filesystem to the glade/campaign

### LINKED ISSUE #266: Convert all references of /glade/p to /glade/campaign
The `/glade/p` filesystem is being retired at the end of 2023. It is being replaced by `/glade/campaign`

Many of the config files reference data on `/glade/p`. The files or directories  they reference must be moved to `/glade/campaign` and the references updated accordingly.

### PR DESCRIPTION
**Pull request by jim-p-w**
### Description:
  The /glade/p filesystem is going away at the end of 2023.
  All files to be retained must be moved to the /glade/campaign filesystem.

  This issue only changes references to files/directories in the code/config files, actual movement of the data was handled separately.

  **Note** I will defer merging this PR until after [PR 268](https://github.com/NCAR/MPAS-Workflow/pull/268) is merged, as there may be some conflicts in .yaml files

  ### Issue closed #266

Changes to be committed:
	modified:   bin/Forecast.csh
	modified:   config/jedi/ObsPlugs/retrieveObsErrors/trainAll.csh
	modified:   initialize/data/Model.py
	modified:   initialize/data/Observations.py
	modified:   scenarios/3denvar_O30kmIE60km_WarmStart.yaml
	modified:   scenarios/3denvar_O30kmIE60km_WarmStart_ABEI.yaml
	modified:   scenarios/3denvar_O30kmIE60km_WarmStart_SpecifiedEnsemble.yaml
	modified:   scenarios/3dhybrid-allsky_O30kmIE60km_SpecifiedEnsemble_VarBC.yaml
	modified:   scenarios/3dhybrid_O30kmIE60km_SpecifiedEnsemble.yaml
	modified:   scenarios/3dhybrid_O30kmIE60km_SpecifiedEnsemble_VarBC.yaml
	modified:   scenarios/3dhybrid_O30kmIE60km_SpecifiedEnsemble_VarBC_iasi.yaml
	modified:   scenarios/ABIAHI120km3denvar.yaml
	modified:   scenarios/IASI120km3denvar.yaml
	modified:   scenarios/IASI30kmIE60km3denvar.yaml
	modified:   scenarios/defaults/externalanalyses.yaml
	modified:   scenarios/defaults/firstbackground.yaml
	modified:   scenarios/defaults/observations.yaml
	modified:   scenarios/defaults/staticstream.yaml
	modified:   scenarios/defaults/variational.yaml
	modified:   scenarios/eda_OIE60km_WarmStart.yaml
	modified:   scenarios/getkf_OIE60km_WarmStart.yaml
	modified:   scenarios/letkf2D_OIE60km_WarmStart.yaml
	modified:   scenarios/letkf_OIE60km_WarmStart.yaml
	modified:   test/testinput/3denvar_O30kmIE60km_WarmStart.yaml

#### Tier 1:
 - [x] 3dvar_OIE120km_WarmStart
 - [x] 3denvar_OIE120km_IAU_WarmStart
 - [x] 3dvar_OIE120km_ColdStart
 - [x] 3dvar_O30kmIE60km_ColdStart
 - [x] 3denvar_O30kmIE60km_WarmStart
 - [x] eda_OIE120km_WarmStart
 - [x] getkf_OIE120km_WarmStart
 - [x] ForecastFromGFSAnalysesMPT

#### Tier 2 (optional):
 - [ ] GenerateGFSAnalyses
 - [ ] GenerateObs

### CODE REVIEW

**Comment by junmeiban** [comment]
Hi @jim-p-w 
This is my first time to try workflow on derecho by using your PR. I use `tcsh`. 
After I launch ` ./Run.py scenarios/3denvar_OIE120km_WarmStart_IAU.yaml`
I can only see “MPAS-Workflow” directory and no others. 
I noticed the following information after I launch the scenario:
```
Cycle: ./submit.csh
module: Command not found.
module: Command not found.
module: Command not found.
./submit.csh (INFO): Generating the scenario-specific MPAS-Workflow directory
```
And same thing happened when I test https://github.com/NCAR/MPAS-Workflow/pull/268
Am I missing something? Any idea of it? 
Many thanks,
Junmei

@junmeiban The README file has been updated to explain how to configure cylc8, which is used on derecho.

@junmeiban The README has been updated with instructions for saving cylc run data in the user scratch space.

**Comment by liujake** [comment]
I think it is better to merge this one first, then #268 next.

File: initialize/data/Observations.py [hunk 1]
```diff
@@ -82,12 +82,12 @@ class Observations(Component):
     'GDASObsErrtable': ['/glade/work/guerrett/pandac/fixed_input/GSI_errtables/HRRRENS_errtable_10sep2018.r3dv', str],
```
Comment (ibanos90) [inline initialize/data/Observations.py]: Can we copy this to campaign as well?

Comment (liujake) [reply initialize/data/Observations.py]: I just did. it is now under /glade/campaign/mmm/parc/liuz/pandac_common/fixed_input/GSI_errtables.

Comment (jim-p-w) [reply initialize/data/Observations.py]: I will update the reference to point to the /glade/campaign location.

Comment (jim-p-w) [reply initialize/data/Observations.py]: Path has been updated.

**Review by liujake — APPROVED** [review]
Had a quick scan. Looks good to me!

**Review by ibanos90 — APPROVED** [review]
All changes look good. Thanks @jim-p-w

**Comment by junmeiban** [comment]
> Hi @jim-p-w This is my first time to try workflow on derecho by using your PR. I use `tcsh`. After I launch ` ./Run.py scenarios/3denvar_OIE120km_WarmStart_IAU.yaml` I can only see “MPAS-Workflow” directory and no others. I noticed the following information after I launch the scenario:
> 
> ```
> Cycle: ./submit.csh
> module: Command not found.
> module: Command not found.
> module: Command not found.
> ./submit.csh (INFO): Generating the scenario-specific MPAS-Workflow directory
> ```
> 
> And same thing happened when I test #268 Am I missing something? Any idea of it? Many thanks, Junmei
> 
> @junmeiban The README file has been updated to explain how to configure cylc8, which is used on derecho.
> 
> @junmeiban The README has been updated with instructions for saving cylc run data in the user scratch space.

@jim-p-w Many thanks for fixing my problem and updated README file

**Review by junmeiban — APPROVED** [review]

**Comment by liujake** [comment]
@jim-p-w Please merge when you are ready. This should be merged first before the 4DEnVar one.

**Pull request merged**